zppy/templates/default.ini

```diff
@@ -226,6 +229,8 @@ ts_num_years_ref = integer(default=5)
 # Name of the `[ts]` subtask for "enso_diags"/"qbo"/"area_mean_time_series"/"streamflow" runs
 # Required for run_type="model_vs_model" "enso_diags"/"qbo"/"area_mean_time_series"/"streamflow" runs
 ts_subsection = string(default="")
+# Required for "tropical_subseasonal" run
+ts_daily_subsection = string(default="")
```

Comment:
I would list these alphabetically, but I guess it doesn't really matter. I suppose I should have been naming the subsection parameters as `subsection_...` so they'd group together alphabetically naturally.

Reply:
to have `subsection` at the end seems to be more natural. I think we can keep it as is, same as the other comment below.